src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -152,6 +156,18 @@ else if(Objects.equals(loansProfile, "Fixed")) {
         Period::asJson);
     }
 
+    if (alternateRecallReturnInterval != null) {
+      if (recalls == null) {
+        recalls = new JsonObject();
+      }
+      putIfNotNull(recalls, "alternateRecallReturnInterval", alternateRecallReturnInterval,
+        Period::asJson);
+    }
+
+    if (recalls != null) {
+      put(recalls, "allowRecallsToExtendOverdueLoans", allowsAlternateRecallReturnInterval);
```

Comment:
I think the variable and the property should have the same name. Given the property in storage is `allowRecallsToExtendOverdueLoans`, then I think the variable should also be.
```suggestion
      put(recalls, "allowRecallsToExtendOverdueLoans", allowRecallsToExtendOverudeLoans);
```

Reply:
This has been changed as you suggested.